### DISCUSSION TITLE
[Task]: patchcoreensemble

### DISCUSSION THREAD

**Discussion by 2324773711** [open-edge-platform/anomalib#2149] [Q&A]
### What is the motivation for this task?

How are the features of different backbone networks combined in PatchCoreEmSemble? cancat or add or what?

### Describe the solution you'd like

no solution!

**Comment by samet-akcay** [comment]
Can you elaborate a bit more? Which `PatchCoreEmSemble` model are you interested in specifically? Maybe some links could help?

**Reply by 2324773711** [reply]
https://arxiv.org/pdf/2106.08265v2--PatchCore paper  Table 4, PatchCoreEmsemble there are three Backbone.DenseN-201&RNext-101&WRN-101.How did they combination.like concat or add,or other ways?

**Reply by abc-125** [reply]
I think they just append it to the list of all features, but you can wait [for the reply in the original repository](https://github.com/amazon-science/patchcore-inspection/issues/103) to be sure.

**Reply by alexriedel1** [reply]
They minmax scale the scores and take the mean after
https://github.com/amazon-science/patchcore-inspection/blob/fcaa92f124fb1ad74a7acf56726decd4b27cbcad/bin/run_patchcore.py#L114-L118

**Comment by 2324773711** [comment]
步步
***@***.***



&nbsp;My question is how the features of the three different backbone(DenseN-201, RNext-101, WRN-101) networks in Table 4 are combined in the ablation experiment of the PatchCore paper.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "openvinotoolkit/anomalib"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月25日(星期二) 下午5:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [openvinotoolkit/anomalib] [Task]: patchcoreensemble (Discussion #2149)





 
They minmax scale the scores and take the mean after
 https://github.com/amazon-science/patchcore-inspection/blob/fcaa92f124fb1ad74a7acf56726decd4b27cbcad/bin/run_patchcore.py#L114-L118
